grunt/tasks/npm-react-addons.js

```diff
@@ -76,15 +85,20 @@ function buildReleases() {
     pkgData.name = pkgName;
 
     grunt.file.mkdir(destDir);
+    var link = info.docs ? info.docs : 'addons';
+    link = `https://facebook.github.io/react/docs/${link}.html`;
     fs.writeFileSync(path.join(destDir, 'index.js'), generateSource(info));
     fs.writeFileSync(path.join(destDir, 'package.json'), JSON.stringify(pkgData, null, 2));
     fs.writeFileSync(path.join(destDir, 'LICENSE'), license);
     fs.writeFileSync(path.join(destDir, 'PATENTS'), patents);
     fs.writeFileSync(
       path.join(destDir, 'README.md'),
-      '# ' + pkgName + '\n\n' +
-      'This package provides the React ' + k + ' add-on. See ' +
-      'http://facebook.github.io/react/docs/addons.html for more information.\n'
+      `
+# ${pkgName}
+
+This package provides the React ${k} add-on.
+
+See <${link}> for more information.`.substr(1)
```

Comment:
nit: `slice` is easier to understand and you can't mix it up with `substring`. I hate substr/substring. You can also ignore this.